src/objects/featured-row/featured-row.scss

```diff
@@ -0,0 +1,42 @@
+@use "../../compiled/tokens/scss/breakpoint";
+@use "../../compiled/tokens/scss/size";
+@use '../../mixins/fluid';
+@use '../../mixins/media-query';
+@use "../../mixins/ms";
+
+.o-featured-row {
+  display: grid;
+  @include fluid.grid-gap(
+    breakpoint.$s,
+    breakpoint.$xl,
+    size.$spacing-gap-fluid-min,
+    size.$spacing-gap-fluid-max
+  );
+
+  @media (min-width: breakpoint.$l) {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, max-content);
+  }
+}
+
+.o-featured-row__heading {
+  @media (min-width: breakpoint.$l) {
+    grid-column: 1 / 2;
+    grid-row: 1 / 2;
+  }
+}
+
+.o-featured-row__content {
+  @media (min-width: breakpoint.$l) {
+    grid-column: 2 / 4;
+    grid-row: 1 / 3;
+  }
+}
+
+.o-featured-row__action {
+  @media (min-width: breakpoint.$l) {
+    align-self: end;
+    grid-column: 1 / 2;
+    grid-row: 2 / 3;
+  }
+}
```

Comment:
Did you consider using `grid-template-areas` to simplify these layout styles?

An example:

```scss

.o-featured-row {
  grid-template-areas:
    'heading' // Each line represents a row
    'content' // Each keyword represents a column
    'action';
  grid-template-rows: repeat(3, minmax(0, auto));

  @media (min-width: breakpoint.$l) {
    // At large breakpoints we update our grid layout, so we also update
    // our areas.
    grid-template-areas:
      'heading content content'
      '. content content' // The dot is just a placeholder to say "leave this empty"
      'action content content';
    grid-template-columns: repeat(3, 1fr);
    grid-template-rows: minmax(0, auto) 1fr minmax(0, auto);
  }
}

// Now all our layout is defined in one place, all that's left is to tell the
// grid which area to put every child element

.o-featured-row__heading {
  grid-area: heading;
}

.o-featured-row__content {
  grid-area: content;
}

.o-featured-row__action {
  grid-area: action;
}
```